doc/user/content/sql/alter-role.md

```diff
@@ -9,15 +9,10 @@ menu:
 `ALTER ROLE` alters the attributes of an existing role.
 
 {{< warning >}}
-Roles in Materialize are currently limited in functionality. In the future they
-will be used for role-based access control. See GitHub issue {{% gh 677 %}}
-for details.
-{{< /warning >}}
-
-{{< warning >}}
-RBAC is under development: currently no role attributes or privileges will be
-considered when executing statements, although these attributes are saved and
-will be considered in a later release.
+Role-Based Access Control is under development {{% gh 11579 %}}. Currently, no
```

Comment:
```suggestion
Role-Based access control is under development {{% gh 11579 %}}. Currently, no
```

Reply:
Made it "Role-based access control" here and elsewhere.